<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StitcherDialog</class>
 <widget class="QDialog" name="StitcherDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>560</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Save</set>
   </property>
  </widget>
  <widget class="QWidget" name="canvasPlaceholder" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>661</width>
     <height>591</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="sourceImagesView">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>0</y>
     <width>130</width>
     <height>561</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
